### PR TITLE
Add event functions

### PR DESCRIPTION
**Pull request by MaikKlein**
Fixes https://github.com/MaikKlein/ash/issues/174

### CODE REVIEW

File: ash/src/device.rs [hunk 1]
```diff
@@ -256,6 +256,36 @@ pub trait DeviceV1_0 {
...
+            allocation_callbacks.as_raw_ptr(),
+            &mut event,
+        );
+        match err_code {
+            vk::Result::SUCCESS => Ok(event),
+            _ => Err(err_code),
+        }
+    }
+    unsafe fn get_event_status(&self, event: vk::Event) -> vk::Result {
+
```
Comment (Ralith) [inline ash/src/device.rs]: Stray whitespace

File: ash/src/device.rs [hunk 2]
```diff
@@ -256,6 +256,36 @@ pub trait DeviceV1_0 {
...
+            create_info,
+            allocation_callbacks.as_raw_ptr(),
+            &mut event,
+        );
+        match err_code {
+            vk::Result::SUCCESS => Ok(event),
+            _ => Err(err_code),
+        }
+    }
+    unsafe fn get_event_status(&self, event: vk::Event) -> vk::Result {
```
Comment (Ralith) [inline ash/src/device.rs]:
```suggestion
    unsafe fn get_event_status(&self, event: vk::Event) -> VkResult<bool> {
```

File: ash/src/device.rs [hunk 3]
```diff
@@ -256,6 +256,36 @@ pub trait DeviceV1_0 {
...
+
+        self.fp_v1_0().get_event_status(self.handle(), event)
+    }
+
+    unsafe fn set_event(&self, event: vk::Event) -> vk::Result {
```
Comment (Ralith) [inline ash/src/device.rs]:
```suggestion
    unsafe fn set_event(&self, event: vk::Event) -> VkResult<()> {
```

File: ash/src/device.rs [hunk 4]
```diff
@@ -256,6 +256,36 @@ pub trait DeviceV1_0 {
...
+        self.fp_v1_0().set_event(self.handle(), event)
+    }
+
+    unsafe fn reset_event(&self, event: vk::Event) -> vk::Result {
```
Comment (Ralith) [inline ash/src/device.rs]:
```suggestion
    unsafe fn reset_event(&self, event: vk::Event) -> VkResult<()> {
```

**Comment by MaikKlein** [comment]
I think we need to update appveyor to a newer version.

**Comment by MaikKlein** [comment]
r+

**Pull request merged**